### ISSUE TITLE
[Bug]: Components that support errors don't show the errors and colouring when using livewire's Volt.

### ISSUE THREAD

**Issue by ChrisRiddell** [tasteui/tasteui#116]
### What Happened

When using Normal Livewire with say using an input component when an error is thrown the input box turns red and you get the error message under it. but when using the same component on Livewire's Volt package you don't get the error message or the red colouring.

### How to Reproduce

Install Laravel Breeze with Livewire Volt class, install TallStackUI, on the login blade just replace the email input component with

```php
<x-input
                type="email"
                label="Email Address"
                wire:model="email"
                required
                autofocus
                autocomplete="username"
            />
```

then login with incorrect email you will get no error, but if you do the same on a normal Laravel install with Auth set up via the doc's with Livewire installed normally it works.

### Livewire Version

3.2.6 + Volt 1.6.0

### TallStackUi Version

1.2.0

### PHP Version

8.3

### Operation System

macOS

**Comment by devajmeireles** [comment]
Hey, @ChrisRiddell ! Thanks for your report.

I was unable to reproduce this issue. Which Laravel version your project is using? Please check all my attempts below. On the left side of each screenshot, you can see the input in red, which indicates the validation error.

1. Volt API Class with form:
![CleanShot 2023-12-05 at 22 38 56](https://github.com/tallstackui/tallstackui/assets/60591772/fc6bb820-d1bc-4408-ba3e-6cdb710abc77)

2. Volt API Class without form:
![CleanShot 2023-12-05 at 22 39 46](https://github.com/tallstackui/tallstackui/assets/60591772/48edd9b6-02c2-4260-a770-2d5acd294033)

3. Volt Functional API with form:
4. ![CleanShot 2023-12-05 at 22 47 15](https://github.com/tallstackui/tallstackui/assets/60591772/57fe236c-0f5d-422a-88e1-1cf6efb1f0d4)

5. Volt Functional API without form:
![CleanShot 2023-12-05 at 22 43 51](https://github.com/tallstackui/tallstackui/assets/60591772/9de5d007-feda-4452-924f-ea0049e3360f)

My environment is:
- Laravel 10.35
- Livewire 3.2.6
- Volt 1.6

Try to reproduce the same steps mentioned above. Feel free to share your code with us in a public GitHub repository, so we can check it in the code.

**Comment by ChrisRiddell** [comment]
<img width="1236" alt="Screenshot 2023-12-06 at 12 15 12 pm" src="https://github.com/tallstackui/tallstackui/assets/109709314/446bd50b-75fd-4aa9-a486-5dc69efc8898">

My environment:

Laravel 10.35
Livewire 3.2.6
Volt 1.6
Herd 1.3.1

Test Repo https://github.com/ChrisRiddell/TestRepo

**Comment by devajmeireles** [comment]
@ChrisRiddell ,

Thanks for sharing the repository. I found the issue and will submit a pull request to fix this.

The problem is that Laravel Breeze with Volt sets the authentication method to throw an exception when authentication fails linked to `email` when it should be `form.email`. When I changed this, the expected behavior happened successfully:

![CleanShot 2023-12-05 at 23 35 05](https://github.com/tallstackui/tallstackui/assets/60591772/7673aca0-37af-4329-a193-d1fd776d4dd7)
